pwndbg/chain.py

```diff
@@ -32,9 +32,9 @@ def get(address, limit=LIMIT, offset=0, hard_stop=None, hard_end=0):
     Returns:
         A list representing pointers of each ```address``` and reference
     """
-    limit = int(limit)+1
+    limit = int(limit)
```

Comment:
The docs for `get` say:
> `limit(int): number of valid pointers`
However if limit is 0 it will increment it to 1 and try to dereference the address once, which is unwanted.